### ISSUE TITLE
Cut v1.6.0 Release on Jan 4th

### ISSUE THREAD

**Issue by negz** [crossplane/crossplane#2764]
### Steps

This issue can be closed when we have:

- [x] Created the [release branch][release-branch].
- [x] Created and merged an empty commit to the `master` branch ([PR][empty-commit-pr]).
- [x] Run the [Tag workflow][rc-tag-workflow] on the `master` branch with the next release candidate tag ([Tag][rc-tag]).
- [x] Updated the current release version in the [Crossplane docs website repo].
- [x] Updated the release branch reaching EOL with docs removal directive.
- [x] Updated all version information in the documentation on the relevant release branch.
- [x] Run the Tag workflow on the relevant release branch with the proper release version.
- [x] Run the CI workflow on the release branch and verified that the tagged build version exists on the [releases.crossplane.io] `build` channel.).
- [x] Run the Configurations workflow on the release branch and verified  that version exists on [registry.upbound.io] for all getting started packages.
    - [x] `xp/getting-started-with-aws`
    - [x] `xp/getting-started-with-with-with-vpc`
    - [x] `xp/getting-started-with-azure`
    - [x] `xp/getting-started-with-gcp`
- [x] Run the Promote workflow with channel `stable` on the release branch and verified that the tagged build version exists on the [releases.crossplane.io] `stable` channel.
- [x] Published a [new release] for the tagged version, with the same name as the version and descriptive release notes.
- [x] Updated the [releases table] in the `README.md` on `master`.
- [x] Ensured that users have been notified of the release on all communication channels:
    - [x] Slack
    - [x] Twitter

[release-branch]: https://github.com/crossplane/crossplane/tree/release-1.6
[empty-commit-pr]: https://github.com/crossplane/crossplane/pull/2765
[rc-tag-workflow]: https://github.com/crossplane/crossplane/actions/runs/1580397169
[rc-tag]: https://github.com/crossplane/crossplane/tree/v1.7.0-rc.0
[website-pr]: https://github.com/crossplane/crossplane.github.io/pull/130

### Notes

[releases.crossplane.io]: https://releases.crossplane.io
[registry.upbound.io]: https://cloud.upbound.io/browse
[new release]: https://github.com/crossplane/crossplane/releases/new
[releases table]: https://github.com/crossplane/crossplane#releases
[Crossplane docs website repo]: https://github.com/crossplane/crossplane.github.io

This release was originally due on Dec 21st per our 8 week schedule, but I'd like to delay to avoid releasing during a week where many of our maintainers and community are likely to be on vacation.

**Comment by hasheddan** [comment]
Currently blocked on #2789

**Comment by negz** [comment]
https://github.com/crossplane/crossplane/pull/2804 should also block this release. https://github.com/crossplane/crossplane/pull/2786 does not block but would be nice to have.

**Comment by negz** [comment]
I got up to the docs pin PR (https://github.com/crossplane/crossplane/pull/2810) but there's no one around to review right now. I may check back in later and continue the release, otherwise I think it's fine for @muvaf to pick it back up in his morning.

**Comment by negz** [comment]
v1.6.0 tag workflow: https://github.com/crossplane/crossplane/actions/runs/1656367214

**Comment by negz** [comment]
v1.6.0 CI workflow: https://github.com/crossplane/crossplane/actions/runs/1656368587

**Comment by negz** [comment]
v1.6.0 configurations workflow: https://github.com/crossplane/crossplane/actions/runs/1656369777

**Comment by negz** [comment]
Configurations:

* https://cloud.upbound.io/registry/xp/getting-started-with-aws-with-vpc/v1.6.0
* https://cloud.upbound.io/registry/xp/getting-started-with-aws/v1.6.0
* * https://cloud.upbound.io/registry/xp/getting-started-with-gcp/v1.6.0
* * https://cloud.upbound.io/registry/xp/getting-started-with-azure/v1.6.0

**Comment by negz** [comment]
Releases:

* https://releases.crossplane.io/build/release-1.6/v1.6.0/
* https://releases.crossplane.io/stable/v1.6.0/

**Comment by negz** [comment]
New release: https://github.com/crossplane/crossplane/releases/tag/v1.6.0

**Comment by negz** [comment]
Slack announcement: https://crossplane.slack.com/archives/CEFQCGW1H/p1641350585127700

**Comment by muvaf** [comment]
Thank you @negz !
